airbyte-commons-protocol/src/main/java/io/airbyte/commons/protocol/migrations/AirbyteMessageMigrationV1.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.protocol.migrations;
+
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.version.AirbyteProtocolVersion;
+import io.airbyte.commons.version.Version;
+import io.airbyte.protocol.models.v0.AirbyteMessage;
+import jakarta.inject.Singleton;
+
+/**
+ * Placeholder AirbyteMessage Migration from v0 to v1
+ */
+@Singleton
+public class AirbyteMessageMigrationV1 implements AirbyteMessageMigration<AirbyteMessage, io.airbyte.protocol.models.AirbyteMessage> {
```

Comment:
is it intentional to upgrade to `io.airbyte.protocol.models.AirbyteMessage` rather than `io.airbyte.protocol.models.v1.AirbyteMessage`?

Reply:
Yes, it is due to the fact that Java connectors and Platform share common libraries that are currently not versioning friendly.

In order to use `io.airbyte.protocol.models.v1.AirbyteMessage`, we need to update pretty much all the protocol objects used in the platform to `io.airbyte.protocol.models.v1.*` which forces us to currently bump some of the Java connectors to use `v1` objects as well. (see the slack thread from yesterday).

The idea here is to take a two step approach https://github.com/airbytehq/airbyte/issues/20037 and https://github.com/airbytehq/airbyte/issues/20038 rather than a giant change to bump platform to v1 and connectors to v0.

Reply:
Yes, the idea here is to have a two step change, enabling v1 on the default objects allows us to have a smaller PR focused on the actual changes to enable v1.
I will have another PR to switch the objects to v1.